src/openMVG_Samples/features_repeatability/CMakeLists.txt

```diff
@@ -1,12 +1,17 @@
-
-ADD_EXECUTABLE(openMVG_sample_main_repeatability_dataset main_repeatability_dataset.cpp)
-TARGET_LINK_LIBRARIES(openMVG_sample_main_repeatability_dataset
+add_executable(openMVG_sample_main_repeatability_dataset main_repeatability_dataset.cpp)
+target_link_libraries(openMVG_sample_main_repeatability_dataset
   openMVG_image
   openMVG_features
   openMVG_matching
   openMVG_system
   openMVG_multiview
   vlsift
-  stlplus)
+  ${POPSIFT_LIBRARIES}
```

Comment:
Yes in this case, since it's a executable adding PUBLIC it's just a quirk. In the internal modules instead it should be always specified, but this is a major PR that involves other refactorings that will be done soon or later. :-)